devops/ansible/roles/girder/library/girder.py

```diff
@@ -896,7 +897,7 @@ def unjsonify(a):
         try:
             a = json.loads(a)
             # pass-through to below
-        except ValueError:
+        except json.JSONDecodeError if sys.version_info >= (3, 5) else ValueError:
```

Comment:
This isn't necessary, as a `json.JSONDecodeError` is a subclass of `ValueError`.

Reply:
While true, I see no arm in catching that specific exception. It would also be consistent with how other calls to json.loads are handled.

Reply:
Yeah, my intent was to capture the most specific exception possible, but perhaps this is overkill. I do anticipate dropping support for Python 3.4 at some point in the medium-term, at which point `sys.version_info >= (3, 5)` would simplify to `six.PY3`.

I'm on the fence, so if you think simplicity is more important, I'd be fine with dropping most of this change.

Reply:
It is ugly and hard to maintain to have each spot where we try to catch JSON decoding errors have this sort of conditional.  If we want the specificity, then, much like six, we should have an abstracted JSONDecodeError which we can catch instead (where we'd define it to be this expression and then individual sections of code can just catch the abstracted error).